sample/Assets/Scripts/AuthenticatedScript.cs

```diff
@@ -399,6 +406,33 @@ public void CancelZkGetBalance()
         ZkGetBalanceCanvas.gameObject.SetActive(false);
     }
 
+    public async void GetZkTransactionReceiptStatus()
+    {
+        try
+        {
+            ShowOutput($"Getting zkEVM transaction receipt...");
+            TransactionReceiptResponse response = await passport.ZkEvmGetTransactionReceipt(ZkGetTransactionReceiptHash.text);
+            ShowOutput($"Transaction receipt status: {response.status}");

```

Comment:
Instead of displaying the returned status, can you show a proper text instead? if `response.status == 0x1`, show 'Success' otherwise, show 'Error'.